### DISCUSSION TITLE
Add Beatmap Description

### DISCUSSION THREAD

**Discussion by Layendan** [ppy/osu#29510] [Ideas] [answered]
Somehow I can't find someone else who requested this feature. Are there any plans atm to add beatmap descriptions to the in-game beatmap screen?

web:
![image](https://github.com/user-attachments/assets/6dbcd4b2-d41f-4503-a1fa-6d064ce1cba3)

osu:
![image](https://github.com/user-attachments/assets/7734a386-f1ac-417b-88aa-a744c0ca36fe)

edit: I'm guessing this change would also allow for user about me's to be displayed on the player info screen

same with the discussion tab to praise/leave feedback, although that should prbly be another discussion thread...

**Comment by bdach** [comment, marked as answer]
Description will be painful because bbcode from the good old days is still wedged in there. Maybe it'll happen but I dunno.

**Reply by Layendan** [reply]
Yeah I see the issue now, best I could find was this library https://github.com/haileys/BBsharp. But that converts it to html which I don't think the osu-framework can handle + might introduce security issues (not a security expert, but seems like a point of entry)

**Reply by Joehuu** [reply]
There's also already an issue opened for this: https://github.com/ppy/osu/issues/8023
